src/part-guide/concurrency.md

```diff
@@ -1,12 +1,12 @@
 # Concurrent programming
 
-The goal of this chapter is to give you a high-level idea of how async concurrency works and how it is different from concurrency with threads. I think it is important to have a good mental model of what is going on before getting in to the practicalities, but if you're the kind of person who likes to see some real code first, you might like to read the next chapter or two and then come back to this one.
+The goal of this chapter is to give you a high-level idea of how async concurrency works and how it is different from concurrency with threads. Forming a good mental model of what is going on will help before getting in to the practicalities. If you're the kind of person who likes to see some real code first, you might like to read the next chapter or two and then come back to this one.
```

Comment:
I think the first change is fine, but prefer to leave the 'but' since the last clause/sentence is in opposition to the previous one